newrelic-weaver/src/main/java/com/newrelic/weave/utils/WeaveUtils.java

```diff
@@ -182,7 +182,9 @@ public final class WeaveUtils {
     private static int getRuntimeMaxSupportedClassVersion() {
         try {
             double jvmSpecVersion = Double.valueOf(System.getProperty("java.specification.version"));
-            if (jvmSpecVersion >=11) {
+            if (jvmSpecVersion >= 17) {
+                return 61;
+            } else if (jvmSpecVersion >= 11) {
```

Comment:
We are going to do something like below in place of the continued branch logic. This way we will also get 19 (and future non-lts and lts versions as well)

```
if version >= 11
  return version + 44;

```
Do you want to make that change? Or I can just pull this into something local and change it.